contrib/Dockerfile

```diff
@@ -1,10 +1,48 @@
-# This file is used to add the nightly Dgraph binaries and assets to Dgraph base
-# image.
+# This file builds the dgraph binary and dgraph docker image.
 
-# This gets built as part of release.sh. Must be run from /tmp/build, with the linux binaries
-# already built and placed there.
+# This gets built as part of our CD pipeline. Must be run through Makefile.
 
-FROM ubuntu:20.04
+FROM --platform=linux/amd64 ubuntu:20.04 AS build
```

Comment:
maybe we could use the golang images instead of downloading Go ourselves

Reply:
I considered that - but the issue is that the golang images use alpine linux, not ubuntu, which would not necessarily be a problem in itself.  The main issue is the alpine linux uses the musl c library instead of glibc, and they are not compatible.  See https://stackoverflow.com/questions/33382707/alpine-musl-vs-glibc-are-they-supposed-to-be-compatible

Reply:
I see debian based images, but I understand if you want to do it yourself. We should add a note to the Dockerfile about this too that we can't use golang docker images as our base image